iso.nix

```diff
@@ -0,0 +1,8 @@
+# This module defines a small NixOS installation CD.  It does not
```

Comment:
A relict of debugging?

Reply:
Ohhh... sorry. Good catch! I'm updating the PR